docker/Makefile

```
@@ -80,6 +80,27 @@ deploy-dev:
 	$(dc) stop app && $(dc) up -d app
 
 
+deploy-stage:
+	# - build section
+	# preserve the dev image from repository
+	$(d) tag fidals/stb:dev fidals/stb:dev_temp
+	$(dcp) stop app-stage
+	$(dcp) rm -f app-stage
```

```suggestion
	$(dcp) rm -fs app-stage
```
It will stop and remove the container